client/src/main/java/com/distkv/client/DefaultDistkvClient.java

```diff
@@ -2,9 +2,13 @@
 
 import com.distkv.asyncclient.DefaultAsyncClient;
 import com.distkv.asyncclient.DistkvAsyncClient;
+import com.distkv.common.utils.FutureUtils;
+import com.distkv.rpc.protobuf.generated.DistkvProtocol.DistkvResponse;
 
 public class DefaultDistkvClient implements DistkvClient {
 
+  private static final String typeCode = "X";
```

Comment:
Why set tycode here?

Reply:
Because other typecodes are in the corresponding proxy, drop and expire are public, no proxy is used. They also need typecode, so here it is.